src/components/v5/shared/Button/ButtonLink.tsx

```diff
@@ -0,0 +1,92 @@
+/* eslint-disable react/button-has-type */
+import React, { FC, PropsWithChildren } from 'react';
+import { useIntl } from 'react-intl';
+import clsx from 'clsx';
+
+import { ButtonLinkProps } from './types';
+import SpinnerLoader from '~shared/Preloaders/SpinnerLoader';
+import styles from './Button.module.css';
+import Link from '../Link';
+import ButtonContent from './ButtonContent';
+
+const displayName = 'v5.ButtonLink';
+
+const ButtonLink: FC<PropsWithChildren<ButtonLinkProps>> = ({
+  mode = 'primarySolid',
+  size = 'default',
+  children,
+  disabled = false,
+  loading = false,
+  title,
+  text,
+  textValues,
+  className,
+  isFullRounded = false,
+  isFullSize,
+  iconName,
+  iconSize = 'tiny',
+  isIconRight,
+  ...rest
+}) => {
+  const { formatMessage } = useIntl();
+
+  const titleText =
+    typeof title === 'string' ? title : title && formatMessage(title);
```

Comment:
I know it's not your code but please make title and texts as `React.ReactNode` as we use it in other places